xrviz/tests/test_control.py

```diff
@@ -10,4 +11,8 @@ def control(data):
 
 def test_control_initial(control):
     tabs = [tab.name for tab in control.tabs]
-    assert tabs == ['Set Coords', 'Variables', 'Axes', 'Style']
+    if cartopy_geoviews_installed():
```

Comment:
Only one of these will ever be triggered in a given test run.

Reply:
Since this is dependent on presence on optional dependencies, only one of these will be triggered.

Reply:
Yes. My point here was, I'm not sure how useful this is as a test. More typical might be to skip if the module import fails, and only check the True branch.